src/chrome/content/rules/openDesktop-mismatches.xml

```diff
@@ -1,150 +0,0 @@
-<!--
-	For rules that are on by default,
-	see openDesktop.xml.
-
--->
-<ruleset name="openDesktop.org (mismatches)" default_off="mismatched">
-
-	<target host="arch-stuff.org" />
-	<target host="www.arch-stuff.org" />
-
-	<target host="beryl-themes.org" />
-	<target host="www.beryl-themes.org" />
-
-	<target host="box-look.org" />
-	<target host="www.box-look.org" />
-
-	<target host="cli-apps.org" />
-	<target host="www.cli-apps.org" />
-
-	<target host="debian-art.org" />
-	<target host="www.debian-art.org" />
-
-	<target host="e17-stuff.org" />
-	<target host="www.e17-stuff.org" />
-
-	<target host="ede-look.org" />
-	<target host="www.ede-look.org" />
-
-	<target host="eyeos-apps.org" />
-	<target host="www.eyeos-apps.org" />
-
-	<target host="frugalware-art.org" />
-	<target host="www.frugalware-art.org" />
-
-	<target host="gentoo-art.org" />
-	<target host="www.gentoo-art.org" />
-
-	<target host="gnomefiles.org" />
-	<target host="www.gnomefiles.org" />
-
-	<target host="gnome-help.org" />
-	<target host="www.gnome-help.org" />
-
-	<target host="gnome-look.org" />
-	<target host="www.gnome-look.org" />
-
-	<target host="gtk-apps.org" />
-	<target host="www.gtk-apps.org" />
-
-	<target host="java-apps.org" />
-	<target host="www.java-apps.org" />
-
-	<target host="kde-apps.org" />
-	<target host="www.kde-apps.org" />
-
-	<target host="kde-files.org" />
-	<target host="www.kde-files.org" />
-
-	<target host="kde-help.org" />
-	<target host="www.kde-help.org" />
```

Comment:
This domain is still under construction. It's probably safer to wait before adding it.